packages/disploy/src/structs/PartialGuildMember.ts

```diff
@@ -12,7 +16,10 @@ export class PartialGuildMember extends Base {
 	 * The member's guild avatar hash.
 	 */
 	public avatar!: string | null;
-
+	/**
+	 * The user the member belongs to.
+	 */
+	public user!: Partial<APIUser>
```

Comment:
Unessarary, while yes most of the code base uses the `!:` syntax, we're migrating away from it and no new code should be using it.
```suggestion
	public user: Partial<APIUser>
```